CHANGELOG.md

```diff
@@ -2,14 +2,18 @@
 
 ## [Unreleased]
```

Comment:
does this part get automagically updated?

Reply:
yea